common/scylla_image_setup

```diff
@@ -30,6 +30,9 @@ if __name__ == '__main__':
             run('systemctl daemon-reload', shell=True, check=True)
             run('systemctl enable var-lib-systemd-coredump.mount', shell=True, check=True)
             run('systemctl start var-lib-systemd-coredump.mount', shell=True, check=True)
+            # some distro has fstrim enabled by default, since we are using XFS with online discard, we don't need fstrim
+            run('if [[ `systemctl is-active fstrim.timer` == "active" ]]; then systemctl disable fstrim.timer; fi', shell=True, check=True)
```

Comment:
`is-active` command has a return code. So, you can just do `systemctl is-active -q fstrim.timer && ...`

Reply:
your'e right, didn't notice the `-q`, when called without it the return code is 0 either way.

Reply:
fixed